README.md

```diff
@@ -0,0 +1,40 @@
+Valkey RFC
+==========
+
+This is a collection of feature proposals and descriptions of changes to Valkey
+that require some more details than just the text in a pull request or an issue.
+It is loosely inspired by RFCs and by Python's enhancement proposals (PEP).
+
+Each feature or larger topic is described in a markdown file that's named in
+uppercase and ends in `.md`. These files are not formally numbered, but we use
+the pull request number that added an RFC to refer to the change. For example,
+this description in the README.md file was written in RFC #1.
+
+Workflow
+--------
+
+An RFC starts off as a pull request. It's reviewed for formatting, style and
+consistency. Then the proposal is merged. This doesn't mean that the feature is
+approved for inclusion in Valkey. It's still just a proposal.
+
+Each file has a status like "draft", "suggested", "approved" or "rejected". The
+possible statuses are yet to be decided.
```

Comment:
I think we should decide that now. "Draft", "Published", "Approved", and "Rejected".

Reply:
Yes. So let's define what these mean and choose terms that are unambigous.

- Draft = non yet merged PR (like this one). I don't think there's any reason to actually spell this out in the document itself. It's enough that the PR is not yet merged.
- Published = RFC merged to this repo but no decision made. PEPs seem to use Draft for this and they accept changes in this stage. I think Published can be interpreted as if the feature has been published, but it's just the RFC. Can we find a better term for this? I propose "Proposed". This is the stage where most of the discussion happens, as in Request For Comments. Perhaps we don't need to spell out a status at all. An RFC without a status is implicitly in this status.
- Approved = Specification approved by core team, awaiting implementation. Minor changes can still be requested in the actual implementation PR. (PEPs use "Accepted" for this.)
- Rejected = Not going to be implemented.

The PEP process has one more status "Final", meaning the feature has been implemented, merged, released. We should update an RFC to mention in which Valkey version it was released and add links to the implementation PR.

Reply:
Should we add the list of states and what they mean here? Right now it's unclear what informational is unless you go look it up in the official IETF documentation.

Reply:
I ended up just adding Proposed, Approved and Rejected. I imagined they can't be misunderstood, but they probably can. I'll define them.